buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -436,6 +472,15 @@
       "allowOverride": true,
       "isSecret": true
     },
+    "TransportFeedAccountName": {
```

Comment:
Can you please name these "AzureBlobFeedAccountName" and "AzureBlobFeedAccessToken"